packages/blaze/view.js

```diff
@@ -490,8 +490,9 @@ Blaze._expand = function (htmljs, parentView) {
 
 Blaze._expandAttributes = function (attrs, parentView) {
   parentView = parentView || currentViewIfRendering();
-  return (new Blaze._HTMLJSExpander(
+  const expanded = (new Blaze._HTMLJSExpander(
     {parentView: parentView})).visitAttributes(attrs);
+  return expanded || {};
 };
```

Comment:
@radekmie this is the part that can be reset, because it's covered by your feature, right?

Reply:
Please double check it, but yes - it's covered in #460.